released/packages/coq-library-undecidability/coq-library-undecidability.1.0.1+8.16/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+version: "1.0.1+8.16"
+maintainer: "forster@cs.uni-saarland.de"
+homepage: "https://github.com/uds-psl/coq-library-undecidability/"
+dev-repo: "git+https://github.com/uds-psl/coq-library-undecidability/"
+bug-reports: "https://github.com/uds-psl/coq-library-undecidability/issues"
+authors: ["Yannick Forster"
+          "Dominique Larchey-Wendling"
+          "Andrej Dudenhefner"
+          "Edith Heiter"
+          "Marc Hermes"
+          "Johannes Hostert"
+          "Dominik Kirst"
+          "Mark Koch"
+          "Fabian Kunze"
+          "Gert Smolka"
+          "Simon Spies"
+          "Dominik Wehr"
+          "Maximilian Wuttke"]
+
+license: "CeCILL-2.1"
```

Comment:
```suggestion
license: "CECILL-2.1"
```
(Also, just to check you're aware of how strong copyleft affects reusability...)

Reply:
We are vaguely aware via the Zulip discussions that happened, but never discussed this properly. I can easily imagine relincensing to something better in the future.

Reply:
Since I don't want to give the wrong idea, CECILL-2.1 is definitely an improvement over regular GPL - you can actually combine CECILL-2.1 with CECILL-B/C and distribute the (compiled) results. The main problem people might see with copyleft of your library is that they have to relicense their projects under either CECILL-2.1 or GPL if they use the library.